cpp/arcticdb/util/buffer.hpp

```diff
@@ -309,11 +309,17 @@ class VariantBuffer {
     }
 
     [[nodiscard]] BufferView view() const {
```

Comment:
Have we got a repro for this? It seems like something that should absolutely never happen

Reply:
It's a use after move in the enterprise replication tooling